service/controller/managementcluster/controller.go

```diff
@@ -73,7 +73,7 @@ func NewController(config ControllerConfig) (*Controller, error) {
 		c := controller.Config{
 			K8sClient: config.K8sClient,
 			Logger:    config.Logger,
-			Name:      project.Name() + "-control-plane-controller",
+			Name:      project.Name() + "-management-cluster-controller",
```

Comment:
We will need to be careful here as we will need to remove the finalizers

Reply:
oh gosh, almost forgot about those. I guess we can keep a tiny `control-plane-controller` which only delete finalizers.

Reply:
Is there a way in operatorkit to have a controller that only removes finalizers without adding them?

Reply:
I don't know

Reply:
added legacy finalizer resource, which takes care of removing the old finalizers.